server/athenian/api/openapi/openapi.yaml

```diff
@@ -7320,3 +7376,235 @@ components:
       - from
       - to
       type: object
+    CodeCheckMetricsRequest:
+      additionalProperties: false
+      description: Request for calculating metrics on top of code check runs (CI) data.
+      example:
+        for:
+        - repositories:
+          - github.com/athenianco/athenian-webapp
+          - github.com/athenianco/athenian-api
+          commit_authors:
+          - - github.com/vmarkovtsev
+            - github.com/dpordomingo
+        - repositories:
+          - github.com/athenianco/metadata
+          - github.com/athenianco/athenian-api
+          commit_authors:
+          - - github.com/vmarkovtsev
+            - github.com/se7entyse7en
+        metrics:
+        - chk-suites-count
+        date_from: 2020-01-23
+        date_to: 2021-01-24
+        granularities:
+        - all
+        account: 1
+      properties:
+        for:
+          description: |-
+            Sets of developers and repositories for which to calculate the metrics.
+            The aggregation is `AND` between repositories and developers.
+            The aggregation is `OR` inside both repositories and developers.
+          items:
+            $ref: '#/components/schemas/ForSetCodeChecks'
+          type: array
+        metrics:
+          description: Requested metric identifiers.
+          items:
+            $ref: '#/components/schemas/CodeCheckMetricID'
+          type: array
+        date_from:
+          description: Date from when to start measuring the metrics.
+          format: date
+          type: string
+        date_to:
+          description: Date up to which to measure the metrics.
+          format: date
+          type: string
+        timezone:
+          $ref: '#/components/schemas/TimeZone'
+        granularities:
+          items:
+            $ref: '#/components/schemas/Granularity'
+          type: array
+        quantiles:
+          $ref: '#/components/schemas/Quantiles'
+        account:
+          description: Session account ID.
+          type: integer
+        split_by_check_runs:
```

Comment:
> option to get the distribution by number of checks run in the suites 
This way means that we will have to make two separate API calls to fetch the CI metrics if we want both overall values and the distribution by suite size.

Reply:
Distribution by number of checks run in the suites (aka suite size) will be used only for:
- number of suites
- suite run time